demo/src/source-info.js

```diff
@@ -231,9 +233,27 @@ const covidTiffInfo = {
   initialViewState: {
     zoom: -7,
     target: [50000, 50000]
-  },
+  }
+};
+
+const omeZarr = {
   url:
-    'https://vitessce-demo-data.storage.googleapis.com/test-data/12448_G1HR_Mesh003.ome.tif'
+    'https://vitessce-demo-data.storage.googleapis.com/test-data/9822151.zarr',
+  dimensions: [
+    // TODO: Having the actual dimensions breaks the UI components currently
+    // ome_zarr images are all (t, c, z, y, x)
+    { field: 'c', type: 'nominal', values: [0] },
+    { field: 'y', type: 'quantitative', values: null },
+    { field: 'x', type: 'quantitative', values: null }
+  ],
+  selections: [{ c: 0 }],
+  isPublic: true,
+  isPyramid: true,
+  description: 'IDR 9822151.zarr - SARS-CoV-2 in human instestinal cells',
```

Comment:
I think we should put `(OME-ZARR)` in parenthesis after the name to match the convention on the rest of the images and make it clear to users what they're looking at.

Reply:
Perhaps we should also make the two descriptions or the COVID-19 images line up more as well so it's clear that they are the same image.